templates/dqlite/index.html

```diff
@@ -0,0 +1,198 @@
+                    <br />
+                    It uses <a href="https://github.com/canonical/raft">C-Raft</a>, an optimized Raft implementation in C, to gain high-performance transactional consensus and fault tolerance while preserving SQlite’s outstanding efficiency and tiny footprint.
+                </p>
+                <div class="p-cta-block">
+                    <a href="/dqlite/docs" class="p-button--positive">Read the Dqlite docs</a>
+                    <a href="https://github.com/canonical/dqlite/releases" class="p-button">Download Dqlite</a>
+                </div>
+                <p>Install on Ubuntu (bleeding edge):</p>
+                <div class="p-code-snippet">
+                    <pre class="p-code-snippet__block--icon"><code>sudo add-apt-repository -y ppa:dqlite/dev && sudo apt install dqlite-tools</code></pre>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="p-section">
+        <hr class="is-fixed-width p-rule" />
+        <div class="row--50-50">
+            <div class="col">
+                <h2>Key features</h2>
+            </div>
+            <div class="col">
+                <ul class="p-list--divided">
+                    <li class="p-list__item is-ticked">Fully asynchronous network and disk I/O</li>
+                    <li class="p-list__item is-ticked">Comprehensive test suite for correctness</li>
+                    <li class="p-list__item is-ticked">Benchmarked for memory footprint and network efficiency</li>
+                    <li class="p-list__item is-ticked">Persistent storage to disk for transaction logs</li>
+                    <li class="p-list__item is-ticked">Fast recovery for system restore</li>
+                    <li class="p-list__item is-ticked">Stable Golang client and a documented wire protocol for other languages</li>
+                    <li class="p-list__item is-ticked">Runs on ARM, X86, POWER and IBM Z architectures</li>
+                </ul>
+            </div>
+            <div class="p-image-container--cinematic is-cover">
+                {{ image(url="https://assets.ubuntu.com/v1/7888ed93-key-features.jpg",
+                                alt="",
+                                width="2464",
+                                height="1027",
+                                hi_def=True,
+                                attrs={"class": "p-image-container__image"},) | safe
+                }}
+            </div>
+        </div>
+    </section>
+
+    <section class="p-section">
+        <hr class="is-fixed-width p-rule" />
+        <div class="u-fixed-width">
+            <div class="p-section--shallow">
+                <h2>Why use Dqlite</h2>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-start-large-4">
+                <div class="row">
+                    <hr class="p-rule--muted u-hide--small" />
+                    <div class="col-4 col-medium-2">
```

Comment:
A col-4 offset means you're only left with 9 cols, but the sizes defined here and in the following row add up to 12 respectively. The hr needs to be moved above the nested row as well for this to work correctly 
```suggestion
                <hr class="p-rule--muted u-hide--small" />
                <div class="row">
                    <div class="col-3 col-medium-2">
```